auto-cpufreq.conf-example

```diff
@@ -101,6 +101,9 @@ turbo = auto
 # enable thresholds true or false
 #enable_thresholds = true
 #
+# whether to check if thresholds are valid (true or false). (THIS SHOULD BE LEFT AS-IS IN MOST CASES)
```

Comment:
What does "(THIS SHOULD BE LEFT AS-IS IN MOST CASES)" mean in this case? Left uncommitted, or left as is it's in example, which is `true`?

If that's the case I would rephrase it like:

`(THIS SHOULD BE LEFT AS-IS IN MOST CASES, e.g: true)`

Reply:
Let's also make sure these changes are reflected everywhere else where it's written, e.g: auto-cpufreq.conf-example.nix